src/core/Breadcrumb/Breadcrumb.baseStyles.ts

```diff
@@ -1,40 +1,38 @@
 import { css } from 'styled-components';
-import { withSuomifiTheme, TokensAndTheme } from '../theme';
+import { defaultThemeTokens as theme } from '../theme';
 import { cssValueToString } from '../../utils/css';
 import { nav, list, listItem, font } from '../theme/reset';
 
-export const baseStyles = withSuomifiTheme(
-  ({ theme }: TokensAndTheme) => css`
-    ${nav({ theme })}
-    ${font({ theme })('bodyTextSmall')}
+export const baseStyles = css`
+  ${nav(theme)}
+  ${font(theme)('bodyTextSmall')}
     background-color: ${theme.colors.whiteBase};
```

Comment:
Indentation doesn't match with others for the background-color.

Tried locally to change it to as others and ran linters and prettier; all seemed happy also then. 

_I remember we had similar things earlier also, but then there was something that linter/prettier did change it back to wrong indentation(?)_

Reply:
Will fix, this are left untouched by linters/prettier.

Reply:
Fixed all I could find.